pkg/config/config.go

```diff
@@ -247,6 +249,22 @@ func (p *Package) LicenseExpression() string {
 	return licenseExpression
 }
 
+// Returns array of ExtractedLicensingInfos formed from the data in
+// the copyright structs found in the conf.
+func (p *Package) LicensingInfos(WorkspaceDir string) (map[string]string, error) {
+        licenseInfos := make(map[string]string)
+        for _, cp := range p.Copyright {
+                if cp.LicensePath != "" {
+                        content, err := os.ReadFile(filepath.Join(WorkspaceDir, cp.LicensePath))
+                        if err != nil {
+                                return nil, fmt.Errorf("failed to read licensepath %q: %w", cp.LicensePath, err)
```

Comment:
@imjasonh  If we can't read the file, the build fails. This happens at the point that spdx generation is called, so like pipelines can download/extract/move things about.